src/common/utils.ts

```diff
@@ -1,6 +1,7 @@
 import fs from 'fs'
 import detectIndent from 'detect-indent'
 import { nullOnNotfound } from './FileUtils'
+const JSON = require('comment-json');
```

Comment:
Can't es import be used? And please remove semicolon. See also failing build: https://travis-ci.org/github/Bessonov/set-project-references/builds/766817256

Reply:
oh weird, when i used it previously i thought they didn't have types, maybe they added. i'll change that as well.

Reply:
oh shoot and semicolon. sorry that's a hard habit to break.

Reply:
may i suggest adding a .prettierrc config and prettier as a devDep? it can automatically take care of things like semicolon removal, so that contributors like me don't accidentally format your files improperly

Reply:
(it is highly opinionated formatter though, so you have to be ok with some if it's unchangeable decisions)

Reply:
ATM you can run `pnpm run lint -- --fix` to fix most of issues. I'm not a fan of prettier or something like husky, but, well, if you want to introduce it, I'm fine with it.